api-docs/streaming-api.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Streaming API
+category: 62fbabf8d9095e057cc1cd2c
+---
+The Trackunit Streaming API is a near-realtime, low latency API, that enables customers to create their own data lake and services, leveraging Trackunit as Telematics Data Service Provider. This document describes the overall design and how to connect to the Streaming API.
+
+> 📘 Available by request
+>
+> The Streaming API is only available by request as an add-on service. Customers need to be on an "Insight" subscription level and set-up costs/API license pricing will be handled by the  account manager. Once commercials are agreed upon Trackunit support can be contacted to set up the Streaming API functionality.
+
+## Design
+
+The Trackunit Streaming API is not a REST API, but instead implemented as a [Kafka](https://kafka.apache.org/)-based solution, backed by [Confluent Cloud](https://confluent.cloud/) and AWS infrastructure, in order to be highly available and performant. When a customer signs up for access to the Streaming API, a Kafka topic is created and a [Flink](https://flink.apache.org/) job starts forwarding data, related to the customer's machines, to that topic. Data is then available for the customer to consume. The data is kept on the topic for 14 days.  
+Only the data related to the customer's machines will be made available for the customer.
+
+Data is available in the Streaming API with low latency after it enters the Trackunit Iris platform. There can be delays between the telematic devices (or 3rd party integrations such as ISO feeds) and the Iris platform due to network connectivity or similar. The common case is however that data is available with near-realtime delays.
+
+## Data Formats and Compatibility
+
+Data is available in two formats:
+
+- **Machine Insights-based**: High-level semantic information about machines, normalized and cleansed. Immediately ready to ingest into a data-lake, or for use in AI-based analytics.
+- **RAW data**: As a direct replacement for the classic Unit APIs, this format is a direct migration path from those.
```

Comment:
Perhaps we should mentioned something similar as with CANMessages vs. MI - that MI stream should be preferred to RAW?

Reply:
Is there any downside for the customer to have both streams?

Reply:
> Perhaps we should mentioned something similar as with CANMessages vs. MI - that MI stream should be preferred to RAW?

In the next paragraph there's "We recommend the Machine Insights based topic for any future integrations." 

> Is there any downside for the customer to have both streams?

I don't know. Is RAW data published in separate topic that has to be configured first? I guess that's slightly more work for us.
I guess it's additional work for customer too - they would have to consume both topics and match the data on their own.